tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -78,15 +79,15 @@ def __init__(self, context: str, sandbox: Optional[str] = None):
 class SerializationOptions(Dict[str, Any]):
     def __init__(
             self,
-            max_dom_depth: Optional[int] = None,
-            max_object_depth: Optional[int] = None,
-            include_shadow_tree: Optional[str] = None
+            max_dom_depth: Union[Optional[int], Undefined] = UNDEFINED,
```

Comment:
Something looks wrong here. Similar to the other arguments it should still be optional for both `int` or `Undefined`. Or am I wrong. Maybe @sadym-chromium can have a look at it as well.

```suggestion
            max_dom_depth: Optional[Union[int, Undefined]] = UNDEFINED,
```

Reply:
https://github.com/web-platform-tests/wpt/blob/master/tools/webdriver/webdriver/bidi/modules/browsing_context.py#L234

Reply:
For my understanding the types `Union[Optional[int], Undefined]` and `Optional[Union[int, Undefined]]` are equal, as `Optional` is a union of `type` and `None`. So both would be resolved to `Union[int, Undefined, None]`. I can be wrong here, as not that familiar with Python types.